src/pages/Home/index.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import ButtonCreateList from 'components/ButtonCreateList';
-import CheckToken from 'components/CheckToken';
+import FormJoinList from 'components/FormJoinList';
 
 const Home = () => (
   <section className="Home">
     <ButtonCreateList />
-    <CheckToken />
+    <FormJoinList />
```

Comment:
Excellent name change, more descriptive.

Reply:
Yes, it's easier to understand 😄 .

Reply:
Nice, thanks for the change!

I'd suggest `JoinListForm` as it reads more naturally.